Framework/Algorithms/test/ReflectometryReductionOne2Test.h

```diff
@@ -465,7 +465,7 @@ class ReflectometryReductionOne2Test : public CxxTest::TestSuite {
     alg.setProperty("ThetaIn", 25.0);
     MatrixWorkspace_sptr outLam = runAlgorithmLam(alg, 12);
 
-    TS_ASSERT_DELTA(outLam->x(0)[0], 0.934991, 1e-6);
+    TS_ASSERT_DELTA(outLam->x(0)[0], 0.934992, 1e-6);
```

Comment:
I'm surprised you had to update the expected values at the level of tolerance.

Reply:
yes, but it's very minor. One would have to ask to the guy who wrote this tests if 1e-6 was really a physically significant tolerance